[2023-12-10 19:42:53,336] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-05-27T00:00:00+00:00 [queued]>
[2023-12-10 19:42:53,484] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-05-27T00:00:00+00:00 [queued]>
[2023-12-10 19:42:53,490] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:42:53,498] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:42:53,504] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:42:53,809] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-05-27 00:00:00+00:00
[2023-12-10 19:42:53,824] {standard_task_runner.py:52} INFO - Started process 3215 to run task
[2023-12-10 19:42:53,862] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-05-27T00:00:00+00:00', '--job-id', '901', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpx2yl06z3', '--error-file', '/tmp/tmp7ud12o19']
[2023-12-10 19:42:53,903] {standard_task_runner.py:80} INFO - Job 901: Subtask segunda_tarea
[2023-12-10 19:42:54,727] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-05-27T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:42:55,941] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-05-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-27T00:00:00+00:00
[2023-12-10 19:42:55,952] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:42:55,959] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-12-10 19:42:56,100] {subprocess.py:85} INFO - Output:
[2023-12-10 19:42:56,132] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-12-10 19:42:56,149] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:42:56,351] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230527T000000, start_date=20231210T194253, end_date=20231210T194256
[2023-12-10 19:42:56,576] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:42:57,808] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
